scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -93,6 +106,7 @@ object FixedPathTypedDelimited {
  * If T is a subclass of Product, we assume it is a tuple. If it is not, wrap T in a Tuple1:
  * e.g. TypedTsv[Tuple1[List[Int]]]
  */
+@deprecated("Use TypedText instead")
```

Comment:
should be TypedTextDelimited I think. Also, can we add a date in the since portion (so people can be reminded how long it has been deprecated).